src/components/MarkdownProvider/components/CodeExample/components/SyntaxHighlighter.tsx

```diff
@@ -7,10 +7,10 @@
 
 import React from 'react';
 import ReactSyntaxHighlighter from 'react-syntax-highlighter';
-import dracula from 'react-syntax-highlighter/dist/esm/styles/hljs/dracula';
+import style from 'react-syntax-highlighter/dist/esm/styles/hljs/atom-one-dark';
```

Comment:
[nit] is it useful to name the import `atomOneDarkStyle` or something more descriptive?

Reply:
We're playing with the different options over time. I didn't want to keep chasing down the variable name.